packages/client/hmi-client/src/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -373,9 +374,11 @@ onUnmounted(() => {
 	flex-direction: column;
 }
 
-#notebook-section:deep(main) {
+.notebook-section:deep(main) {
 	gap: var(--gap-small);
 	position: relative;
+	/** TODO: Temporary solution, should be using the default overlay-container padding in tera-drilldown */
```

Comment:
In my opinion, the overlay-container should not have any padding. We should let each drilldown decides how to be displayed.

Reply:
I am leaning towards passing a prop like `removeDefaultPadding` in the case the padding needs to be specified.

Reply:
Updated the comment to mention your opinion